plugins/kubernetes/db/migrate/20160606202534_create_kubernetes_jobs.rb

```diff
@@ -0,0 +1,16 @@
+class CreateKubernetesJobs < ActiveRecord::Migration
+  def change
+    create_table :kubernetes_jobs do |t|
+      t.references :stage, null: false, index: true
+      t.references :kubernetes_task, null: false, index: true
+      t.references :user, null: false, index: true
+      t.references :build
```

Comment:
Just like in `Kubernetes::Release`, first the job is created and then the build, therefore, we cannot set it to be null: false.